libcontainer/dmz/nolibc/Makefile

```diff
@@ -0,0 +1,91 @@
+# SPDX-License-Identifier: GPL-2.0
```

Comment:
Should be removed to avoid confusion (If we are going to merge this PR)

Reply:
I think I prefer to keep it if we _can_ keep it. Otherwise, updating to a newer version of nolibc in the future would be more painful (or we need to make a script that takes that, removes this, maybe something else, etc.).

I think there is value in the simplicity of just a simple c&p.

Reply:
> I think there is value in the simplicity of just a simple c&p.

The problem is that this is legally less simple.
It is not obvious (for non-maintainers) that the GPL-2.0 code is actually not linked with runc.

Reply:
Fixed, then! :)